tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -271,7 +276,7 @@ private async Task SendRequestsAsyncNoWaitForSpans(string url, string body, int
 
         private IImmutableList<MockSpan> WaitForSpans(MockTracerAgent agent, int expectedSpans, string phase, DateTime minDateTime)
         {
-            var urls = new[] { "Health", "Home/Upload", "data" };
+            var urls = new[] { "Health", "Home/Upload", "data", "good", "bad", "void" };
```

Comment:
it should have been `ugly` not `void` :p

Reply:
I think, The Good, The Bad and The Void has a certain ring to it 😛